docs/islandora/rdf-mapping.md

```diff
@@ -1,39 +1,132 @@
+
+- The JSON-LD module will only work with mappings that include a value under `types` (which maps to `rdf:type` - see below, under Structure of an RDF Mapping).
+- The JSON-LD module provides a hook so other modules can alter the entity before it gets mapped. The `islandora` module uses this hook to trigger any "Map URI to Predicate" and "Alter JSON-LD Type" reactions that are configured in Contexts. `islandora_defaults` provides the two Contexts - "All Media" and "Content" - that configure these to occur on Media and Repository Item nodes.
+- The JSON-LD module adds RDF datatypes to the RDF values, and includes a mapping of Drupal field types to RDF datatypes.
+- The JSON-LD module provides a hook to alter its Drupal field type to RDF datatype mapping.
+- The JSON-LD module has a configuration option that can cause the `?_format=jsonld` to be part of, or not part of, the URIs of Drupal objects. On an out-of-the-box claw-playbook, this string is stripped, but by default on a fresh install of the jsonld module, it is not.
+
+
+## RDF Mappings
+In an out-of-the-box claw-playbook, the RDF mappings that exist were loaded from config files, and correspond to the rdf.mapping.[...].yml files located in:
+
+- `[drupal modules directory]/islandora/modules/islandora_core_feature/config/install/` (media and taxonomy terms)
+- `[drupal modules directory]/islandora_defaults/config/install/` (`repository_item` and the `islandora_access` vocabulary)
+- `[drupal web root]/core/profiles/standard/config/install/` (articles, pages, comments, and tags).
+
+Once loaded by modules, configuration .yml files are not live so **editing them will not change the existing configuration**. However, for modules that are Features, it is possible to re-import the changed configuration files at `admin/config/development/features` (todo: link to further reading on Features).
+
+### How to edit an RDF Mapping
+
+Once loaded, RDF mappings can be customized for the needs of a particular site through Drupal's Configuration Synchronization UI at `admin/config/development/configuration`. They can be exported, modified, and re-imported one-at-a-time by choosing the "Single Item" option on the Export/Import tabs.  You can also create new RDF mappings (e.g. for a custom content type) and load them through this interface, by copying an existing mapping and changing the appropriate values.
+
+!!! note "Possible future UI for RDF Mappings"
+    A custom module `rdfui` exists, and is installed-but-not-enabled on boxes provisioned by the claw-playbook. We don't use it because it is very rudimentary and limited to the schema.org vocabulary. There is a project underway to develop a [UI](https://github.com/Islandora-CLAW/CLAW/issues/647) to support RDF mappings to any ontology.
```

Comment:
Despite the presence of a ticket and some initial conversations, I don't know that the UI development is 'underway'. Atleast, I'm not aware of anyone actively working on it.

Reply:
was just copying from a previous version!